sdk/opa.go

```diff
@@ -39,17 +39,17 @@ import (
 // OPA represents an instance of the policy engine. OPA can be started with
 // several options that control configuration, logging, and lifecycle.
 type OPA struct {
-	id           string
-	state        *state
-	mtx          sync.Mutex
-	logger       logging.Logger
-	console      logging.Logger
-	plugins      map[string]plugins.Factory
-	store        storage.Store
-	hooks        hooks.Hooks
-	config       []byte
-	v1Compatible bool
```

Comment:
Do we want to keep this? We'll have a v0Comaptible flag as well.

Reply:
This is an internal implementation detail. We still have the front-facing `sdk.Options.V1Compatible`, which when set to `true` will set the appropriate `OPA.regoVersion` value.